src/System.Net.WebSockets.Client/tests/ClientWebSocketTests.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Text;
+
+using Xunit;
+
+namespace System.Net.WebSockets.Client.Test
+{
+    /// <summary>
+    /// ClientWebSocket unit tests that do not require a remote server.
+    /// </summary>
+    public class ClientWebSocketTest
+    {
+        internal static class AssertExtensions
+        {
+            public static void Throws<T>(Action action, string message)
+                where T : Exception
+            {
+                Assert.Equal(Assert.Throws<T>(action).Message, message);
+            }
+        }
+
+        [Fact]
+        public void ClientWebSocket_Ctor()
+        {
+            var cws = new ClientWebSocket();
```

Comment:
Aside: We had some old tests that were ported that were essentially doing this followed by ```Assert.NotNull(cws);```.  Thank you for not doing that :smile: